pinchito/users.go

```diff
@@ -0,0 +1,56 @@
+package pinchito
+
+
+type TgPinchitoUser struct {
+	TgUsername  string
+	PinId int
+	PinNick  string
+}
+
+var tgPinchitoUsers []TgPinchitoUser
+
+func InitUsers() {
+	tgPinchitoUsers = []TgPinchitoUser{
```

Comment:
I'd like to explore the possibility of loading this as a JSON config file rather than having it here, publicly available

Reply:
where would you put this file?
and how would you load it?